images/macos/provision/utils/utils.sh

```diff
@@ -87,15 +89,11 @@ is_Less_BigSur() {
 verify_http_status_code()
 {
     URL="$1"
-    http_code=$(curl -s -o out.html -w '%{http_code}' $URL;)
+    http_code=$(curl -sL -o out.html -w '%{http_code}' $URL)
 
-    if [[ $http_code -ge 400 ]]; then
-        cat out.html
+    if [[ $http_code -eq 200 ]]; then
```

Comment:
it should be not equal
```suggestion
    if [[ $http_code -ne 200 ]]; then
```